File: server/DTO/ProduitDTO.php
<?php


class ProduitDTO implements JsonSerializable
{
    private $idProduit;
    private $nom;
    private $prix;
    private $idType;

    public function __construct($nom, $prix, $idType)
    {
        $this->nom = $nom;
        $this->prix = $prix;
        $this->idType = $idType;
    }

    public function getIdProduit()
    {
        return $this->idProduit;
    }

    public function setIdProduit($idProduit)
    {
        $this->idProduit = $idProduit;
    }

    public function getNom()
    {
        return $this->nom;
    }

    public function setNom($nom)
    {
        $this->nom = $nom;
    }

    public function getPrix()
    {
        return $this->prix;
    }


    public function setPrix($prix)
    {
        $this->prix = $prix;
    }


    public function getIdType()
    {
        return $this->idType;
    }


    public function setIdType($idType)
    {
        $this->idType = $idType;
    }

    public function jsonSerialize()
    {
        return array(
            'idProduit' => $this->idProduit,
            'nom' => $this->nom,
            'prix' => $this->prix,
            'idType' => $this->idType,
        );
    }
}

File: client/src/main/java/com/sio/pizzeria/DTO/ProduitDTO.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.sio.pizzeria.DTO;


/**
 *
 * @author Lenovo P50
 */
public class ProduitDTO {
    
    private int idProduit;
    private String nomProduit;
    private float prix;
    private int idType;

    public ProduitDTO(String nomProduit, float prix, int idType) {
        this.nomProduit = nomProduit;
        this.prix = prix;
        this.idType = idType;
    }



    public int getIdProduit() {
        return idProduit;
    }

    public void setIdProduit(int idProduit) {
        this.idProduit = idProduit;
    }

    public String getNomProduit() {
        return nomProduit;
    }

    public void setNomProduit(String nomProduit) {
        this.nomProduit = nomProduit;
    }

    public float getPrix() {
        return prix;
    }

    public void setPrix(int prix) {
        this.prix = prix;
    }

    public int getIdType() {
        return idType;
    }

    public void setIdType(int idType) {
        this.idType = idType;
    }

    
    
}
